test/e2e/status.sh

```diff
@@ -0,0 +1,10 @@
+echo full01: $(curl localhost:5701/status -s | jq '.result.sync_info.latest_block_height')
```

Comment:
just a dummy script to keep track of the heights when running e2e. could be run using `bash status.sh`